test/files/neg/forward.scala

```diff
@@ -21,4 +23,13 @@ object Test {
     Console.println("foo");
     def g: Int = f;
   }
+  {
```

Comment:
I'd be remiss as a reviewer if I didn't mention that putting the import in this brace results in smaller diff.

Reply:
Too late now